modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/Actions.tsx

```diff
@@ -43,17 +43,11 @@ export default function Actions({
 	style,
 	url,
 }: IFDSTableProps) {
-	const [creationLanguageId, setCreationLanguageId] = useState<
-		Liferay.Language.Locale
-	>();
-
 	useEffect(() => {
```

Comment:
Hi @juanjofgliferay. We can remove this useEffect since it was only used the API's response to set the creationLanguageId.